docs/source/app-dev/bindings-scala/index.rst

```diff
@@ -0,0 +1,143 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Scala bindings
+##############
+
+The Scala bindings is a client implementation of the *Ledger API*
+based on `Akka Streams API <https://doc.akka.io/docs/akka/current/stream/index.html>`_.
+
+Introduction
+============
+
+The Scala bindings library lets you write applications that connect to the Digital Asset distributed ledger using the Scala programming language.
+
+There are two main components:
+
+- Akka Streams-based API
+    The API that you use to send commands to the ledger and receive transactions back.
```

Comment:
Likewise, we probably want to mention a third component, the akka-less API.

Reply:
@stephencompall-DA what is that 3rd component? We have not tested Scala codegen with anything else but akka-based API.

Reply:
you technically can send generated commands using any existing API, doesn't have to be scala akka-based bindings, can be java rx-based api too. But the 3rd component... is something I am not aware of.